tests/unit/serve/runtimes/gateway/grpc/test_grpc_tls.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import grpc
+import pytest
+from docarray import Document
+
+from jina import Client, Flow
+from jina.serve.networking import GrpcConnectionPool
+from jina.types.request.control import ControlRequest
+
+
+@pytest.fixture
+def cert_pem():
+    """This is the cert entry of a self-signed local cert"""
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    return f'{cur_dir}/cert/server.crt'
+
+
+@pytest.fixture
+def key_pem():
+    """This is the key entry of a self-signed local cert"""
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    return f'{cur_dir}/cert/server.key'
+
+
+def test_grpc_ssl_with_flow(cert_pem, key_pem):
+    with Flow(
+        protocol='grpc',
+        ssl_certfile=cert_pem,
+        ssl_keyfile=key_pem,
+    ) as f:
+        os.environ['JINA_LOG_LEVEL'] = 'ERROR'
```

Comment:
If u change this env variable, put it in a `fixture` that after running puts it back to the original value:

```python
@pytest.fixture
def error_log_level():
    old_env = os.environ.get('JINA_LOG_LEVEL')
    os.environ['JINA_LOG_LEVEL'] = 'ERROR'
    yield
    os.environ['JINA_LOG_LEVEL'] = old_env
```

Reply:
Should I changed it there as well ? (this is were I originally took the code) https://github.com/jina-ai/jina/blob/dc8a99addde6f16637cc8439fed4d6208bc6384e/tests/unit/serve/runtimes/gateway/http/test_app.py#L207

Reply:
yes